test/AutoDiff/SILOptimizer/param_thunk_tuple.swift

```diff
@@ -0,0 +1,33 @@
+// RUN: %target-swift-frontend -emit-sil %s | %FileCheck %s
+
+// Verify the result type of a subset parameters thunk matches the declaration:
+//
+// CHECK: // autodiff subset parameters thunk for forward-mode derivative from f(x:)
```

Comment:
I'm not sure if demangled names are printed in all build configurations. Please let me know if it is not the case.
Mangled names are a bit too long and unreadable, but we can check them instead if we have to.